staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/exec.go

```diff
@@ -173,6 +209,10 @@ func (r *roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 	if err != nil {
```

Comment:
this should switch to getting the whole creds set, and passing the whole creds set to `refresh()`

Reply:
Done